README.md

```diff
@@ -90,6 +90,7 @@ Design System | [Uno.Material](https://platform.uno/docs/articles/external/uno.t
 [Navigation](doc/Architecture.md#navigation) | [Chinook.Navigation](https://github.com/nventive/Chinook.Navigation)<br/>[Chinook.BackButtonManager](https://github.com/nventive/Chinook.BackButtonManager)
 [Validation](doc/Validation.md) | [FluentValidation](https://fluentvalidation.net/)
 [App Reviews](doc/Reviews.md) | [ReviewService](https://github.com/nventive/ReviewService)
+[Forced Update](doc/ForcedUpdate) | [UpdateRequiredService](src/app/ApplicationTemplate.Business/ForcedUpdates/UpdateRequiredServiceMock.cs)
```

Comment:
I don't know if we should list this feature here since it's not using a 3rd party library.

Reply:
Yeah, I was not sure about it, but it was in the AC, I guess I will remove it from there

Reply:
I asked him myself, to link it, may not be required, I thought we might have a package in the future. But it's also true that we can remove it.

Reply:
> Yeah, I was not sure about it, but it was in the AC, I guess I will remove it from there

Please strike the AC like ~~that~~.